src/vosk/util.ts

```diff
@@ -0,0 +1,46 @@
+import path from 'path';
+import os from 'os';
+import fs from 'fs';
+import { PLATFORMS, platformPaths } from './helpers';
+
+export const getBaseDLLPath = () => {
+  // Path is different in dev than in production
+  const prodPath =
+    process.env.NODE_ENV === 'development'
+      ? '../../assets/voskDLLs'
+      : '../../../assets/voskDLLs';
+
+  return path.join(__dirname, prodPath, 'lib');
+};
+
+export const getDLLDir = () => {
+  const baseDLLPath = getBaseDLLPath();
+
+  let dllDir;
+  if (os.platform() === PLATFORMS.WINDOWS) {
```

Comment:
Could do a `Record<Platform, string>` for the paths

Reply:
Good idea. My bad for not thinking of this. Done! (Also combined two objects into one for more cleanliness!)